staging/src/k8s.io/apiserver/go.mod

```diff
@@ -12,6 +12,7 @@ require (
 	github.com/evanphx/json-patch v4.12.0+incompatible
 	github.com/fsnotify/fsnotify v1.5.4
 	github.com/gogo/protobuf v1.3.2
+	github.com/google/cel-go v0.12.5
```

Comment:
This dep was added only due to the `ref.Val` reference in the interface. Is it worth losing type safety to prevent adding the dep?

https://github.com/kubernetes/kubernetes/pull/112858/files#diff-56a253b6ac5fca0b2d9e1fd2ecd5a2b9ddf0eb9c2c061beae0ced9493af8a24bR57

Reply:
It should be gone by now. Please rebase.